src/routes/auth/auth.controller.spec.ts

```diff
@@ -99,13 +97,21 @@ describe('AuthController', () => {
         `auth_nonce_${nonceResponse.body.nonce}`,
         '',
       );
+      const expirationTime = faker.date.future();
       const message = siweMessageBuilder()
         .with('address', signer.address)
         .with('nonce', nonceResponse.body.nonce)
+        .with('expirationTime', expirationTime.toISOString())
         .build();
       const signature = await signer.signMessage({
         message: toSignableSiweMessage(message),
       });
+      const expirationTimeInSeconds = getSecondsUntil(expirationTime);
+      // MaxAge does not include the second of expiration
+      const maxAge = expirationTimeInSeconds - 1;
+      // jsonwebtoken sets expiration based on timespans, not exact dates
+      // meaning we cannot use expirationTime directly
+      const expires = new Date(Date.now() + expirationTimeInSeconds * 1_000);
```

Comment:
These intricacies are simplified in [a follow up PR](https://github.com/safe-global/safe-client-gateway/pull/1414).